### ISSUE TITLE
Question About Pretraining Details of Provided ResNet Weights

### ISSUE THREAD

**Issue by ywlywl6** [yhhhli/BRECQ#45]
We would like to ask how these pretrained weights were obtained in terms of training configuration. Would it be possible to share some key details?

We’ve observed that the ResNet weights you provided perform significantly better than the official weights from timm and torchvision v2 in many quantization scenarios.

For example, in the case of ResNet-50 with W8A2, in our tests, the brecq weights you provided achieve 55.75% Top-1 accuracy after quantization, whereas the timm a1 / a2 / a3 weights only reach 0.08% / 0.09% / 0.11% Top-1 respectively, and the torchvision ResNet-50 V2 weights only achieve 12.07% Top-1.

This difference is already extremely significant.

**Comment by ywlywl6** [comment]
中文版问题：想请教一下，这些预训练权重大概是按照什么训练配置得到的，是否方便公开一些关键信息？我们这边观察到，你们提供的 ResNet 权重在很多量化场景下都明显优于 timm 和 torchvision v2的官方权重。以 ResNet-50 的 W8A2 为例，在我们的测试中，你们提供的 brecq 权重量化后可以达到 55.75% Top-1，而 timm 的 a1 / a2 / a3 权重分别只有 0.08% / 0.09% / 0.11% Top-1，torchvision ResNet-50 V2 权重也只有 12.07% Top-1。这种差异已经非常明显
